code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -69,31 +69,151 @@
 	permeability_coefficient = 0.9
 	siemens_coefficient = 0.9
 
+//Special type of gloves. Alt click and you get some special nodrop gloves
+/obj/item/clothing/gloves/heldgloves
+	name = "gloves"
+	/// What type of glove we use for the right hand
+	var/rightglove_path
+	/// What type of glove we use for the left hand
+	var/leftglove_path
+	/// The glove we're currently using in the right hand
+	var/obj/item/weapon/heldglove/rightglove
+	/// The glove we're currently using in the left hand
+	var/obj/item/weapon/heldglove/leftglove
+
+/obj/item/clothing/gloves/heldgloves/Destroy()
+	QDEL_NULL(rightglove)
+	QDEL_NULL(leftglove)
+	return ..()
+
+/obj/item/clothing/gloves/heldgloves/examine(mob/user)
+	. = ..()
+	to_chat(user, "<span class='notice'>Alt-click the gloves when worn to strap them into your hands.")
+
+/obj/item/clothing/gloves/heldgloves/unequipped(mob/unequipper, slot)
+	. = ..()
+	remove_gloves(unequipper)
```

Comment:
no longer usefull

Reply:
Still called on delimbs, clears up the glove nodrop as well as the weapon gloves themselves. They'd just plop on the floor without this